src/App.scss

```diff
@@ -40,7 +40,7 @@ body {
 .main {
   flex: 1 1 auto;
   overflow: hidden;
-  padding: 140px;
+  padding: 140px 0 0 0;
```

Comment:
Padding top?

Reply:
In another pull request it was fix

Reply:
so revert this change to avoid conflicts maybe?